lms/envs/common.py

```diff
@@ -2372,6 +2372,7 @@
 FINANCIAL_REPORTS = {
     'STORAGE_TYPE': 'localfs',
     'BUCKET': 'edx-financial-reports',
+    'CUSTOM_DOMAIN': 'edx-financial-reports.s3.amazonaws.com',
```

Comment:
Also since we already have "BUCKET", perhaps "BUCKET_ROOT" or "BUCKET_DOMAIN" makes more sense?